.github/workflows/veramo-website.yml

```diff
@@ -0,0 +1,20 @@
+name: Trigger veramo-website build
+on:
+  push:
+    branches:
+      - "main"
+      - "italo/fix-docs"
+permissions: write-all
+jobs:
+  dispatch:
+    permissions:
+      contents: write
+      actions: write 
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          curl -X POST \
+            -H 'Authorization: Bearer ${{ secrets.GH_TOKEN }}' \
+            -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/repos/uport-project/veramo-website/actions/workflows/main.yml/dispatches \
```

Comment:
It seems safer to call the repository_dispatch event api:
https://docs.github.com/en/rest/repos/repos#create-a-repository-dispatch-event

I believe that if we do that there won't be a need to mess with permissions either.

Reply:
Done.